sass/_nav-header

```diff
@@ -0,0 +1,103 @@
+.top-nav {
+  display: flex;
+  justify-content:flex-end;
+  align-items: center;
+  margin-top: 2rem;
+  height: 2rem;
+  width: 100%;
+  padding: 1rem;
+}
+
+.icon-nav {
+   display: flex;
+   list-style: none;
+ }
+
+.subscribe {
+  color: $border_white;
+  text-decoration: none;
+  margin-right: 10px;
+}
+
+.fa-circle {
+  color: $border_white;
+  font-size: 5px;
+}
+
+.circle-border {
+  display: flex;
+  flex-direction: column;
+  padding: 1.7rem;
+}
+
+.nav__icons {
+   padding: .2rem;
+   font-size: 2.5rem;
+   color: rgba(130, 146, 150, 0.5);
+ }
+
+#nav-facebook {
+  padding: 1.2rem;
+  background-color: rgb(43, 77, 128);
+  box-shadow: $drop_shadow;
+}
+
+#nav-rss {
+  padding: 1.2rem;
+  background-color: rgb(182, 115, 54);
+  box-shadow: $drop_shadow;
+}
+
+#nav-twitter {
+  padding: 1.2rem;
+  background-color: rgb(64, 124, 153);
+  box-shadow: $drop_shadow;
+}
+
+#nav-youtube {
+  padding: 1.2rem;
+  background-color: $brand_red;
+  box-shadow: $drop_shadow;
+}
+
+.nav-site {
+  display: flex;
+  width: 100%;
+  margin-top: 3rem;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav-item {
+  margin: 10px;
+  width: 15rem;
+  padding: .5rem;
+  text-align: center;
+  color: $border_white;
+  border: 5px $brand_red solid;
+  background-color: $brand_red;
+  box-shadow: $drop_shadow;
+}
+
+.hero {
+  position: relative;
+  width: 100%;
+  border: $border_style;
+  box-shadow: $drop_shadow;
+}
+
+.hero-img {
+  width: 100%;
+}
+
+.logo {
+  position: absolute;
+  top: 10%;
+  left: 2%;
+}
+
+.radio-btns {
+  text-align: center;
+  margin: 1rem auto;
+  transform: scale(2);
```

Comment:
This transform is having ill effects. This doubles the width of this `.radio-btns` and then it makes the page too large to fit the content.